src/components/molecules/OfferDetails/OfferDetails.tsx

```diff
@@ -203,40 +229,52 @@ export const OfferDetails: FC<IOfferDetails> = ({
               >
                 {formatPayment(offer)}
               </Text>
-              {hasPaymentBonuses && (
-                <DetailsList items={[...offer.financialBonuses, ...offer.benefits]} />
-              )}
+              {hasPaymentBonuses && <DetailsList items={[...financialBonuses, ...benefits]} />}
             </Flex>
           </Flex>
-          {!isEmpty(offer.employmentForms) && (
-            <Flex mt="20px" color={getBadgeColor('C3F0E1')}>
-              <Icon name="Agreement" />
-              <Flex flexDirection="column" ml={2} sx={{ overflow: 'hidden' }}>
-                <Text sx={{ fontWeight: 'bold', fontScale: 4, color: 1 }}>Rodzaj umowy</Text>
-                <DetailsList items={offer.employmentForms} />
-              </Flex>
-            </Flex>
-          )}
-          {!isEmpty(offer.workingTimes) && (
-            <Flex mt="20px" color={getBadgeColor('FDF3DA')}>
-              <Icon name="Disposition" />
-              <Flex flexDirection="column" ml={2} sx={{ overflow: 'hidden' }}>
-                <Text sx={{ fontWeight: 'bold', fontScale: 4, color: 1 }}>Wymiar czasu pracy</Text>
-                <DetailsList items={offer.workingTimes} />
-              </Flex>
+          <Flex mt="20px" color={getBadgeColor('C3F0E1')}>
+            <Icon name="Agreement" />
+            <Flex flexDirection="column" ml={2} sx={{ overflow: 'hidden' }}>
+              <Text sx={{ fontWeight: 'bold', fontScale: 4, color: 1 }}>Rodzaj umowy</Text>
+              {!isEmpty(employmentForms) && <DetailsList items={employmentForms} />}
             </Flex>
-          )}
-          {!isEmpty(offer.description) && (
-            <Flex flexDirection="column" mt={4} sx={{ overflow: 'hidden' }}>
-              <Text sx={{ fontWeight: 'bold', fontScale: 4, mb: 1 }}>Szczegóły oferty</Text>
-              <Text fontScale={3}>{offer.description}</Text>
+          </Flex>
+
+          <Flex mt="20px" color={getBadgeColor('FDF3DA')}>
+            <Icon name="Disposition" />
+            <Flex flexDirection="column" ml={2} sx={{ overflow: 'hidden' }}>
+              <Text sx={{ fontWeight: 'bold', fontScale: 4, color: 1 }}>Wymiar czasu pracy</Text>
+              {!isEmpty(workingTimes) && <DetailsList items={workingTimes} />}
             </Flex>
-          )}
+          </Flex>
+
+          <Flex flexDirection="column" mt={4} sx={{ overflow: 'hidden' }}>
+            <Text sx={{ fontWeight: 'bold', fontScale: 4, mb: 1 }}>Szczegóły oferty</Text>
+            {!isEmpty(description) ? (
```

Comment:
to przecież będzie się źle wyświetlało w kandydacie, jak nie będzie description u nas to pokaże się jakiś Tekst Opis, dodatkowa treść oferty itd. a tego nie chcemy.

dodatkowo czemu !isEmpty a potem ternary, niepotrzebna negacja, lepiej dać isEmpty i ternary w odwrotnej kolejności. łatwiej czytać kod.

Reply:
ale opis jest wymagany, więc w teorii nie ma możliwości i sądziłam że nie jest to problem, bo w podglądzie oferty w przypadku braku opisu mamy wyświetlić to co poniżej ;)

Reply:
a no to przepraszam. Czyli nie da się dodać oferty bez description?

Reply:
nope :)